R/pkg/inst/tests/testthat/test_context.R

```diff
@@ -19,21 +19,25 @@ context("test functions in sparkR.R")
 
 test_that("Check masked functions", {
   # Check that we are not masking any new function from base, stats, testthat unexpectedly
-  masked <- conflicts(detail = TRUE)$`package:SparkR`
-  expect_true("describe" %in% masked)  # only when with testthat..
-  func <- lapply(masked, function(x) { capture.output(showMethods(x))[[1]] })
-  funcSparkROrEmpty <- grepl("\\(package SparkR\\)$|^$", func)
-  maskedBySparkR <- masked[funcSparkROrEmpty]
+  # This list should not change!
```

Comment:
just want to make sure people are aware that this list of names shouldn't be added to without considering what it means? if there's a better way to convey that let me know?